bigbluebutton-html5/imports/ui/components/breakout-room/component.jsx

```diff
@@ -210,25 +224,41 @@ class BreakoutRoom extends PureComponent {
     } = this.state;
 
     const roomItems = breakoutRooms.map(breakout => (
-      <div className={styles.content} key={`breakoutRoomList-${breakout.breakoutId}`}>
-        <span aria-hidden>
-          {intl.formatMessage(intlMessages.breakoutRoom, breakout.sequence.toString())}
-          <span className={styles.usersAssignedNumberLabel}>
-            (
-            {getNumUsersByBreakoutId(breakout.breakoutId)}
-            )
-          </span>
-        </span>
-        {waiting && requestedBreakoutId === breakout.breakoutId ? (
-          <span>
-            {intl.formatMessage(intlMessages.generatingURL)}
-            <span className={styles.connectingAnimation} />
+      <div className={styles.breakoutItems}>
+        <div className={styles.content} key={`breakoutRoomList-${breakout.breakoutId}`}>
+          <span aria-hidden>
+            {intl.formatMessage(intlMessages.breakoutRoom, breakout.sequence.toString())}
+            <span className={styles.usersAssignedNumberLabel}>
+              (
+              {getNumUsersByBreakoutId(breakout.breakoutId)}
+              )
+            </span>
           </span>
-        ) : this.renderUserActions(breakout.breakoutId, breakout.sequence.toString())}
+          {waiting && requestedBreakoutId === breakout.breakoutId ? (
+            <span>
+              {intl.formatMessage(intlMessages.generatingURL)}
+              <span className={styles.connectingAnimation} />
+            </span>
+          ) : this.renderUserActions(breakout.breakoutId, breakout.sequence.toString())}
+        </div>
+        <div className={styles.joinedUserNames}>
+          {breakout.joinedUsers
+            .sort(BreakoutRoom.sortById)
+            .filter((value, idx, arr) => !(value.userId === (arr[idx + 1] || {}).userId))
+            .sort(UserListService.sortUsersByName)
```

Comment:
I'm not super crazy about reusing this sort method. If we change the way it functions we've got a hidden dependency here.